src/client/setup.ts

```diff
@@ -27,10 +27,22 @@ function performanceMetrics(): void {
 	);
 }
 
+function platformCSS(): void {
+	const getPlatformClass = (): string => {
+		const ua = navigator.userAgent;
+		if (/android/i.test(ua)) {
```

Comment:
I suspect this might be brittle, but it's a temporary workaround for a bug so 👍 for now.